agent/src/main/java/com/teamscale/jacoco/agent/options/AgentOptionsParser.java

```diff
@@ -156,12 +163,39 @@ private void handleOption(AgentOptions options,
 						value)) {
 			return;
 		}
+   		if (key.startsWith("proxy-") && handleProxyOptions(options, StringUtils.stripPrefix(key, "proxy-"), value, filePatternResolver)){
+				return;
+			}
 		if (handleAgentOptions(options, key, value)) {
 			return;
 		}
 		throw new AgentOptionParseException("Unknown option: " + key);
 	}
 
+	private boolean handleProxyOptions(AgentOptions options, String key, String value, FilePatternResolver filePatternResolver) throws AgentOptionParseException {
+		String httpsPrefix = String.format("%s-", ProxySystemProperties.Protocol.HTTPS);
+		if (key.startsWith(httpsPrefix)
+				&& options.getTeamscaleProxyOptions(ProxySystemProperties.Protocol.HTTPS).handleTeamscaleProxyOptions(StringUtils.stripPrefix(
+				key, httpsPrefix), value)) {
+			return true;
+		}
+
+		String httpPrefix = String.format("%s-", ProxySystemProperties.Protocol.HTTP);
+		if (key.startsWith(httpPrefix)
+				&& options.getTeamscaleProxyOptions(ProxySystemProperties.Protocol.HTTP).handleTeamscaleProxyOptions(StringUtils.stripPrefix(
+				key, httpPrefix), value)) {
+			return true;
+		}
+
+		if(key.equals("password-file")) {
+			Path proxyPasswordPath = filePatternResolver.parsePath(key, value);
+			options.getTeamscaleProxyOptions(ProxySystemProperties.Protocol.HTTPS).proxyPasswordPath=proxyPasswordPath;
+			options.getTeamscaleProxyOptions(ProxySystemProperties.Protocol.HTTP).proxyPasswordPath=proxyPasswordPath;
```

Comment:
Sorry I missed this before. Please use  a setter method for this instead of setting it with the raw value here. 
```suggestion
			Path proxyPasswordPath = filePatternResolver.parsePath(key, value);
			options.getTeamscaleProxyOptions(ProxySystemProperties.Protocol.HTTPS).setProxyPasswordPath(proxyPasswordPath);
			options.getTeamscaleProxyOptions(ProxySystemProperties.Protocol.HTTP).setProxyPasswordPath(proxyPasswordPath);
```